crates/languages/src/rust.rs

```diff
@@ -522,6 +526,14 @@ impl ContextProvider for RustContextProvider {
                     target.target_kind.to_string(),
                 ),
             ]));
+            if let Some(features) = target.required_features {
+                variables.insert(
+                    RUST_BIN_REQUIRED_FEATURES_TASK_VARIABLE.clone(),
+                    format!("--features {}", features.join(",".into())),
+                );
+            } else {
+                variables.insert(RUST_BIN_REQUIRED_FEATURES_TASK_VARIABLE, "".into());
```

Comment:
I'm not entirely happy about this, but couldn't find a better solution for an optional task variable.

Reply:
cc @osiewicz but I think this is as best as we can get now, given that the new always requires this variable in the `"Run {} {} (package: {})",` template.